lib/util/jshint-reporter.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
```

Comment:
Isn't this file more correctly called `jshint-reporter-loader.js`?

I'm also wondering whether this file should be in the `build` directory. Not sure.